openstack/swift-utils/templates/account-caretaker-collect-daemonset.yaml

```diff
@@ -31,7 +35,7 @@ spec:
             name: swift-account-caretaker
         - name: swift-bin
           configMap:
-            name: swift-bin
+            name: swift-bin # NOTE: This comes from the "swift" chart.
```

Comment:
That makes me nervous. Is there any particular reason we just refer to the swift config map and don't get the needed parts in an own cm?